app/actions/websocket.ts

```diff
@@ -51,6 +50,7 @@ import {loadChannelsForTeam, markAsViewedAndReadBatch} from '@actions/views/chan
 import {getPost, getPosts, getPostsAdditionalDataBatch, getPostThread} from '@actions/views/post';
 import {getMe, loadMe} from '@actions/views/user';
 import {GlobalState} from '@mm-redux/types/store';
+import {loadRolesIfNeeded} from '@mm-redux/actions/roles';
```

Comment:
I rather avoid using this action, is not efficient. We ended up replacing where we load roles like this https://github.com/mattermost/mattermost-mobile/blob/master/app/actions/views/user.js#L62 because of the inefficiency on the action

Reply:
Done.